frontend/common/kubeflow-common-lib/projects/kubeflow/src/styles.scss

```diff
@@ -1,8 +1,17 @@
-@use '~@angular/material' as mat;
+@use '@angular/material' as mat;
 @import './kubeflow.css';
 @import './lib/variables.scss';
 
 // always include only once per project
+// TODO(v15): As of v15 mat.legacy-core no longer includes default typography styles.
```

Comment:
Is there an issue for this?

Reply:
Ok good catch for the todo. But It's actually probably most likely irrelevant now. It was introduced during the angular v15 update, which was the update that brought a lot of changes to the components and was the update that flagged a lot of our components as "legacy" which required me to run the migration script to update all the legacy components.
Pre migration script:
![image](https://github.com/StatCan/jupyter-apis/assets/84033116/f866b70a-f04b-4a80-b8b6-6a8374019798)

Post migration script: 
![image](https://github.com/StatCan/jupyter-apis/assets/84033116/4d0d0a2e-685b-49d3-b093-8a7a332df6ad)


If the todo is irelevent now though, it should be removed so I will go and do that.